compiler/modules/print.llvm/llvm.bal

```diff
@@ -772,8 +876,26 @@ function createLine(string[] words, string indent = "") returns string {
     return string:concat(indent, ...parts);
 }
 
+function concat(string... words) returns string {
```

Comment:
This has duplicate functionality with createLine

Reply:
Fixed